app/templates/interstitial.html

```diff
@@ -1 +1,45 @@
-{% include 'questionnaire.html' %}
+{% extends theme('layouts/_survey.html') %}
```

Comment:
I think this would be better solved by using template inheritance and setting a variable for the content of the button text. This way we can avoid duplicating elements of the template across files.